tests/template/test_templater.py

```diff
@@ -60,3 +60,23 @@ def test_render_template(self):
         templater.template_ext = [".txt"]
         rendered_content = templater.render_template(templater.message, context)
         assert rendered_content == "Hello world"
+
+    def test_not_render_literal_value(self):
+        templater = Templater()
+        templater.template_ext = []
+        context = Context({"name": "world"})  # type: ignore
```

Comment:
Why not just use a key that’s defined here?

Reply:
I guess if the test is meant to keep a macro unresolved, it's better to leave the context empty or it can be confusing